llvm/lib/IR/RepoTicket.cpp

```diff
@@ -165,12 +198,58 @@ std::tuple<bool, unsigned, unsigned> generateTicketMDs(Module &M) {
       return updateDigestGONumAndGetDependencies(GO, GOHash, GOIMap, GONum);
     };
     updateDigestUseCallDependencies(&GO, Hash, Visited, GOIMap, Helper);
+    for (const auto GV : GOIMap[&GO].Contributions) {
+      // Record GO's possible memory dependents.
+      GVIMap[GV].insert(&GO);
+    }
+
     MD5::MD5Result Digest;
     Hash.final(Digest);
     set(&GO, Digest);
     Changed = true;
   }
 
+  if (GVIMap.empty()) {
+    return std::make_tuple(Changed, GONum.VarNum, GONum.FuncNum);
+  }
+
+  // Update the GV's digest using the contributions.
+  for (auto &It : GVIMap) {
```

Comment:
The name `It` suggests that it's an iterator but it isn't (it's a GVIMap value_type). Is there a better name?

Reply:
Changed from 'It' to 'Entry'.